### DISCUSSION TITLE
add support for importing icon via url

### DISCUSSION THREAD

**Discussion by nobaraos12** [JulianPrieber/littlelink-custom#252] [Feature requests]
adds support for importing button icons from url

**Comment by JulianPrieber** [comment]
Isn't that exactly what the custom website button does?

**Reply by nobaraos12** [reply]
yes but it doesn't detect icons for google drive folders, so having the ability to import an google drive logo via url would be quite nice

**Reply by JulianPrieber** [reply]
You can just add a Google icon with a custom button.

We are working on a button where you can upload a custom icon, but I can't say when that will eventually come out.

**Reply by nobaraos12** [reply]
> You can just add a Google icon with a custom button.

where can this button be found?

**Reply by JulianPrieber** [reply]
![chrome_AqdyxVgaJ6](https://user-images.githubusercontent.com/60265788/205663279-12ab6710-cea4-482e-b4b8-8ace0af3b369.png)

This block, add your link, then click on customize.

**Reply by nobaraos12** [reply]
didn't realize this was there thank you
